src/Adyen/Service/ResourceModel/Management/MerchantAccount.php

```diff
@@ -13,12 +13,16 @@ class MerchantAccount extends \Adyen\Service\AbstractResource
     protected $allowApplicationInfo = false;
 
     /**
+     * @param array|null $queryParams
      * @return mixed
      * @throws \Adyen\AdyenException
      */
-    public function list()
+    public function list(array $queryParams = null)
     {
         $url = $this->managementEndpoint . "/merchants";
+        if (!empty($queryParams)) {
+            $url .= '?' . http_build_query($queryParams);
+        }
         return $this->requestHttp($url, 'get');
```

Comment:
The `requestHttp` method accepts the params as array|null and takes care of adding it to the request
```suggestion
        return $this->requestHttp($url, 'get', $queryParams);
```

Reply:
curl client does not use the params afterall, better to add it there